src/test/java/com/github/cukespace/arquillian/asciidoctor/AdocGeneratorTest.java

```diff
@@ -9,21 +10,21 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import java.io.File;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.Writer;
+import java.io.*;
 
 @RunWith(Arquillian.class)
 public class AdocGeneratorTest {
+
     @Deployment(testable = false)
     public static Archive<?> app() {
-        return ShrinkWrap.create(WebArchive.class, "app.war")
+        WebArchive war = ShrinkWrap.create(WebArchive.class, "app.war")
                 .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+        System.out.println(war.toString(true));
```

Comment:
Maybe a personal thing but i dont like these dump cause i gind they dont bring much

On a less subjective side: at least use a logger cause otherwise output is mixed with logger outputs